db/migrate/20150212214222_remove_duplicate_services2.rb

```diff
@@ -1,11 +1,16 @@
 class RemoveDuplicateServices2 < ActiveRecord::Migration
   def change
-    duplicate_keys = Mdm::Service.count(group: [:host_id, :port, :proto]).select { |k,v| v >1 }.keys
-    duplicate_keys.each do |keys|
-      duplicate_services = Mdm::Service.where(host_id: keys[0], port: keys[1], proto: keys[2]).order(:created_at)
-      duplicate_services.pop
-      duplicate_services.each(&:destroy)
-    end
+    select_mgr = Mdm::Service.arel_table.project(
```

Comment:
I'm a little confused. This file looks exactly the same as the one above. "20150212214222_remove_duplicate_services2.rb" and "20150112203945_remove_duplicate_services.rb"

It looks like the same migration is run twice? Not sure if this is a mistake or working as expected.